### ISSUE TITLE
index.html "updated" date is incorrect

### ISSUE THREAD

**Issue by Engenia** [Engenia/Engenia#7]
in index.html there is a date at the bottom that denotes the most recent update time. Most pages have this at the top.
In the most recent merge, index.html has not had this updated. It should read something like "19 Dec 2023"

**Comment by mi-hol** [comment]
right, now with the move to git, changes are easy to find via commit history.
Therefore I'd suggest:
 to replace this date with a link to commit history 
or
set/source the date from git via a workflow

**Comment by Engenia** [comment]
The reason for the update note in the webpage is not to keep the developer
abreast of the changes, but for the reader.
I find it greatly annoying when I visit a site that doesn't mention when it
was written. It doesn't fill me with confidence that what I'm reading is up
to date. Hence the date on my pages.

One question re.Markdown.
When you created AltParts.md, does Github automatically generate
AltParts.html?
If so, how frugal is it?
I've seen other programs do this rather poorly.

regards

**Comment by mi-hol** [comment]
> Hence the date on my pages

ok, so we'll go with option 2 "set/source the date from git via a workflow".
I'll manually update the date for now and create a new issue for option 2.



> When you created AltParts.md, does Github automatically generate AltParts.html?

no, I created AltParts.html manually based on advise from [here](https://github.com/zerodevx/zero-md/discussions/99#discussioncomment-7882959)

**Comment by mi-hol** [comment]
https://github.com/Engenia/Engenia/issues/9

**Comment by Engenia** [comment]
My plan for the immediate future is to retain engenia.com.au as the primary
site & use github as the development site. I'll transfer any changes to the
primary & my original local copy, which means I need html to continue for
now.

I may shut down the engenia site at a later date. I'll canvas the forum.

Heading to Queensland for Christmas & won't have easy access to github
until the new year.

Have a Merry Christmas Michael

regards
Errol

**Comment by mi-hol** [comment]
> which means I need html to continue for now.

I'm not entirely sure to understand, could you please elaborate a bit on this (i.e technical issue encountered with local testing of website, etc)?

Thank you and wish a Merry Christmas too!

Regards
Michael

**Comment by Engenia** [comment]
Was only thinking of the ease of updating the engenia site & my own local
one, by simply copying the github zero-md script generated html files to it.


regards
Errol

**Comment by mi-hol** [comment]
> copying the github zero-md script generated html files

Maybe there is a misunderstanding.
The html is only created temporary at runtime "on the fly" while a user views the page, the html created out of md file can't be copied from GitHub. The zero-md script implements that function, from my understanding it should work in every setup independent of GitHub.
As I don't know the details of your local setup to test the website, I'd suggest to try with a local copy from my GitHub fork.
I'd be eager to learn about any technical issues encountered.

**Comment by Engenia** [comment]
I'll try it in the new year when we get back home.

regards
Errol

**Comment by mi-hol** [comment]
@Engenia have you had a chance to try yet?
